utils/airdropWindows.ts

```diff
@@ -31,9 +31,73 @@ export enum WindowStatus {
   LAST_CLAIM = 'LAST_CLAIM',
 }
 
+export const AIRDROP_SUPPORT_QUERY_STRING = 'Nunet Airdrop Support Enquiry';
 export const TOTAL_AIRDROP_TOKENS_STRING = 'Total Airdrop Tokens';
 export const TOTAL_AIRDROPS_STRING = 'Airdrops';
 export const SUCCESSFUL_REGISTRATION_STRING = 'Successfully registered for Airdrop window';
+export const AIRDROP_ELIGIBILITY_STRING = 'Airdrop Eligibility';
+export const AIRDROP_NOT_QUALIFIED_STRING = 'Not Qualified';
+export const AIRDROP_INELIGIBILITY_REASON_STRING = 'Sory. You are not qualified for this Airdrop Window';
+export const AIRDROP_BLOG_POST = 'https://medium.com/nunet/nunet-ntx-token-airdrop-for-agix-token-holders-c13a89b154b7';
+export const AIRDROP_HOW_IT_WORKS_STRING = 'How Nunet Airdrop works';
+export const AIRDROP_TITLE_STRING = 'NuNet Airdrop';
+export const AIRDROP_DESCRIPTION_STRING = 'NuNet is distributing 5% of its total supply of one billion NTX tokens to SingularityNET AGIX token holders. ' +
+'A total of 50 million NTX will be distributed across the four airdrops of 12.5 million tokens each, in January, April, July and October 2022. ' +
+'The October 2022 airdrop will reward only those users who have held AGIX  tokens since the start of the airdrops in January 2022.';
+
+export const HOW_IT_WORKS = [
+  /*{
```

Comment:
@raamb can you remove this commented section if it is not going to use?

Reply:
@rajeevbbqq I kept this as we may need to add it back. Will remove after confirmation from Nunet team